Plugin server crashed: Cannot find module './server'

## Bug description
Hey Posthog team,

when trying to run Posthog locally using the normal `docker compose up` command I always run into the following error message on the `web` service:

```bash
web_1    | [1/4] Resolving packages...
web_1    | success Already up-to-date.
web_1    | Done in 0.48s.
web_1    | yarn run v1.22.10
web_1    | $ posthog-plugin-server
web_1    | internal/modules/cjs/loader.js:883
web_1    |   throw err;
web_1    |   ^
web_1    | 
web_1    | Error: Cannot find module './server'
web_1    | Require stack:
web_1    | - /code/plugins/node_modules/posthog-plugin-server/dist/src/index.js
web_1    | - /code/plugins/node_modules/posthog-plugin-server/bin/posthog-plugin-server
web_1    |     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
web_1    |     at Function.Module._load (internal/modules/cjs/loader.js:725:27)
web_1    |     at Module.require (internal/modules/cjs/loader.js:952:19)
web_1    |     at require (internal/modules/cjs/helpers.js:88:18)
web_1    |     at Object.<anonymous> (/code/plugins/node_modules/posthog-plugin-server/dist/src/index.js:23:18)
web_1    |     at Module._compile (internal/modules/cjs/loader.js:1063:30)
web_1    |     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
web_1    |     at Module.load (internal/modules/cjs/loader.js:928:32)
web_1    |     at Function.Module._load (internal/modules/cjs/loader.js:769:14)
web_1    |     at Module.require (internal/modules/cjs/loader.js:952:19) {
web_1    |   code: 'MODULE_NOT_FOUND',
web_1    |   requireStack: [
web_1    |     '/code/plugins/node_modules/posthog-plugin-server/dist/src/index.js',
web_1    |     '/code/plugins/node_modules/posthog-plugin-server/bin/posthog-plugin-server'
web_1    |   ]
web_1    | }
web_1    | error Command failed with exit code 1.
web_1    | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
web_1    | 
web_1    | 🔴 Plugin server crashed!
web_1    | ⌛️ Waiting 2 seconds before restarting!
```

## Expected behavior
I would expect the service to start normally


## How to reproduce

1. Pull the latest master branch (`9f9516ab8e6097dc088a92c7370ba7d46ce70a1a`)
2. Adjust the `docker-compose.yml`: Add the `SECRET_KEY` env variable
3. Run `docker-compose up`

## Environment

- [ ] PostHog Cloud
- [x] self-hosted PostHog, version/commit: 
   - Master branch: `9f9516ab8e6097dc088a92c7370ba7d46ce70a1a`
   - Image: latest ( `DIGEST:sha256:8ba18719477cf5657b8ef40a13343151db4ee4964fbf6ec6a554b8fb15c5246b`)

## Additional context

I already tried to have a look at the code but could not figure it out. I hope it is not related to my docker setup being corrupt since it worked yesterday morning. But maybe you can also help me pointing that out.

What I tried additionally:

- Go inside the container and remove `/code/plugins/node_modules` + run `yarn install` in `/code/plugins`
- Run `docker-compose -f docker-compose.dev.yml up` -> here only the `worker` service crashes with the same error as above

Hope you can help me out on that since we really want to test out Posthog for our product analytics purposes. It sounds like a promising solution to our problems :) 

### UPDATE
 **Changing the docker image in `docker-compose.yml` to `posthog/posthog:latest-release` (which in this case would be `1.19.0` I guess)  -> WORKED without error**

I still leave the issue open because I guess it might be related to the latest changes to the plugin server

--- a/plugins/package.json
+++ b/plugins/package.json
@@ -4,7 +4,7 @@
     "license": "MIT",
     "private": true,
     "dependencies": {
-        "posthog-plugin-server": "0.6.2"
+        "posthog-plugin-server": "0.6.4"
     },
     "scripts": {
         "start": "posthog-plugin-server"

